Library/Formula/bork.rb

```diff
@@ -0,0 +1,18 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.1.tar.gz"
+  sha256 "01ff60779582d5ec67080d43a6325d3f5f213dd3dbbbd1ba0989e0253bdc73aa"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match "checking: directory #{testpath}/foo\r" \
```

Comment:
> Looks like your editor removed the trailing whitespace too 🙀

Ha! I have a Sublime Text plugin for that if I recall correctly. :grin: 

I've also put the expected output in a variable. I think it looks a bit better since the previous working version. All good?